.github/workflows/pr-test-generated-files.yaml

```diff
@@ -37,15 +37,14 @@ jobs:
             echo "Install GCC on the runner."; 
           fi
 
-      # Note we do not use apt install -y protobuf-compiler` since it is too old
+        # Note we do not use apt install -y protobuf-compiler` since it is too old
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why do we need the repo-token here?

Reply:
Because the action pulls the binaries from a github repo. protoc is only released on github